coursereviews/templates/static_pages/about.html

```diff
@@ -2,4 +2,4 @@
 
 {% block content %}
   <h2>About page</h2>
-{% endblock %}
\ No newline at end of file
+{% endblock %}
```

Comment:
Can you remove this change as well?

You can reset the changes on the branch with `git reset --soft HEAD^1` which will reset the changes made in the past 1 commit (you're currently at `HEAD`). The you can unstage the changes with `git reset HEAD coursereviews/templates/static_pages/about.html` and remove the change entirely with `git checkout -- coursereviews/templates/static_pages/about.html`. Most of these commands are in the help message when you run `git status`.

After all this, you can recommit the changes you want (the first three files) with `git commit -m "Some meaningful commit message about how you access AdminQuota."` and force push the changes with `git push -f origin adminquota`.